taskRequests/style.css

```diff
@@ -0,0 +1,182 @@
+:root {
+  font-family: 'Inter', sans-serif;
+  --color-primary: #1d1283;
+  --color-success: rgba(20, 102, 75, 0.6);
+  --color-error: #da1e28;
+  --color-warn: rgba(199, 129, 18, 0.4);
+  --color-gray-light: #eee;
+  --color-gray: #666;
+  --color-text-light: rgba(0, 0, 0, 0.6);
+  --elevation-1: 0 1px 3px 1px rgba(0, 0, 0, 0.1),
+    0 1px 2px 0 rgba(0, 0, 0, 0.1);
+  --elevation-3: 0px 1px 3px 0px rgba(0, 0, 0, 0.3),
+    0px 4px 8px 3px rgba(0, 0, 0, 0.15);
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+.hidden {
+  display: none !important;
```

Comment:
Use of `!important` is not encouraged.

Reply:
Remove `!important` from `.hidden` class styling